backend/Pipfile

```diff
@@ -77,8 +77,8 @@ python-dotenv = "*"
 dataclasses-json = "*"
 jwcrypto = "*"
 libra-offchainreference = "*"
-client-python-sdk = {git="https://github.com/libra/client-sdk-python.git", editable=true}
 sqlalchemy-paginator = "*"
+diem = "*"
```

Comment:
you may lock to current version. next diem version will include breaking changes that are pushed to diem core codebase, hence blind upgrade will break wallet.